pkg/validation/validation.go

```diff
@@ -75,18 +79,21 @@ func NodeGroups(nodeGroups []nropv1.NodeGroup) error {
 }
 
 // TODO: move it under the validation webhook once we will have one
-func nodeGroupsMachineConfigPoolSelector(nodeGroups []nropv1.NodeGroup) error {
+func nodeGroupPools(nodeGroups []nropv1.NodeGroup) error {
 	for _, nodeGroup := range nodeGroups {
-		if nodeGroup.MachineConfigPoolSelector == nil {
-			return fmt.Errorf("one of the node groups does not have machineConfigPoolSelector")
+		if nodeGroup.MachineConfigPoolSelector == nil && nodeGroup.PoolName == nil {
+			return fmt.Errorf("one of the node groups does not set a pool specifier")
```

Comment:
I'd say "nodegroup missing any pool specifier"
we can do
```
for idx, nodeGroup := range nodeGroups {
    if nodeGroup.MachineConfigPoolSelector == nil && nodeGroup.PoolName == nil {
			return fmt.Errorf("node group %d missing any pool specifier", idx)
    }
```

to perhaps help user a bit